app/scripts/controllers/preferences.test.js

```diff
@@ -39,6 +40,7 @@ describe('preferences controller', function () {
   });
 
   afterEach(function () {
+    process.env.TOKEN_DETECTION_V2 = false;
```

Comment:
I think it could be better to restore the process.env value to its original value. 

Something like.
https://stackoverflow.com/questions/48033841/test-process-env-with-jest

For our case, I think we could use the `beforeAll` and `afterAll` hooks instead of `beforeEach` and `afterEach` since we won't be updating the values in the individual tests.